src/Survey/ItechArt.Survey.WebApp/appsettings.json

```diff
@@ -9,5 +9,30 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "Serilog": {
+    "Using": [],
+    "MinimumLevel": {
+      "Default": "Information",
+      "Override": {
+        "Microsoft": "Fatal",
+        "System": "Fatal"
+      }
+    },
+    "WriteTo": [
+      {
+        "Name": "File",
+        "Args": {
+          "path": "../../Survey/ItechArt.Survey.WebApp/bin/Debug/net6.0/logs/Survey.log",
+          "outputTemplate": "{Timestamp} [{Level:u3}] {Message} {Exception:1} {NewLine:1}"
```

Comment:
1) what does :1 mean?
2) will inner exceptions be logged?
  2.1) raise some exception with inner exception in controller to test it